### ISSUE TITLE
Show Organization and User name- Personalization

### ISSUE THREAD

**Issue by aliciabeckett-gov** [CDCgov/prime-central#114]
**As a user of the app I want to know that I am logged into the right account and submitting data to the right organization so that my data is secure and is being saved correctly**

Users need feedback that they are logged into the correct account for the correct organization. That way they can verify that when they submit data it is linked to them, and they are not accidentally using another user's account. It also gives them peace of mind that they are submitting data for the correct facility/organization.

**Describe the solution you'd like and the specific acceptance criteria to be completed**

Design
- [x] UI for how to display who the logged in user is (personalization for the user)
- [x] UI that shows for which facility/organization the data is being recorded.

Engineering
- [x] Add user level personalization per the design
- [x] Add organization level personalization per the design

**Blockers**
This is blocked by authentication. We need to have some users to know who they are. #102

**Comment by willusds** [comment]
Lower priority, could be Sprint 5
